src/libs/Network/processRequest.js

```diff
@@ -0,0 +1,19 @@
+import HttpUtils from '../HttpUtils';
+import enhanceParameters from './enhanceParameters';
+import * as NetworkEvents from './NetworkEvents';
+
+/**
+ * @param {Object} request
+ * @param {String} request.command
+ * @param {Object} request.data
+ * @param {String} request.type
+ * @param {Boolean} request.shouldUseSecure
```

Comment:
Is this one optional / does is default to `false` ?

```suggestion
 * @param {Boolean} [request.shouldUseSecure]
```

Reply:
Not optional here, but it is here so I can fix that

https://github.com/Expensify/App/blob/ee5d18e2bb47a9d2b71fcca08ae1254b56a95d4c/src/libs/Network/index.js#L317-L320